src/hotspot/share/classfile/dictionary.cpp

```diff
@@ -341,14 +345,82 @@ void Dictionary::add_protection_domain(int index, unsigned int hash,
 }
 
 
-bool Dictionary::is_valid_protection_domain(unsigned int hash,
+inline bool Dictionary::is_valid_protection_domain(unsigned int hash,
                                             Symbol* name,
                                             Handle protection_domain) {
   int index = hash_to_index(hash);
   DictionaryEntry* entry = get_entry(index, hash, name);
   return entry->is_valid_protection_domain(protection_domain);
 }
 
+void Dictionary::validate_protection_domain(unsigned int name_hash,
+                                            InstanceKlass* klass,
+                                            Handle class_loader,
+                                            Handle protection_domain,
+                                            TRAPS) {
+
+  // Now we have to call back to java to check if the initating class has access
```

Comment:
Can you move this comment down about 10 lines?

Reply:
Ok Done.